CHANGELOG.md

```diff
@@ -518,9 +518,7 @@ Version 1.0.1 release of bilby
 Version 1.0 release of bilby
 
 ### Changes
-- Minor bug fixes and typo changes only from 0.6.9, see
```

Comment:
Can we add somewhere in this file that the links were changed retroactively after migration to github?

Reply:
Maybe also link this PR while doing so.

Reply:
I've added a note at the top of the document, let me know what you think.